content/livesync/models/index.textile

```diff
@@ -0,0 +1,203 @@
+---
+title: Setup
+meta_description: "Install the Ably Models SDK to get started with creating data models to keep your frontend in sync with your backend database."
+product: livesync
+languages:
+  - javascript
+---
+
+The Ably Models SDK is a standalone SDK built on "Ably's JavaScript SDK":/getting-started/setup with full TypeScript support. It enables you to easily create live, observable data models in your frontend applications to ensure they remain synchronized with your database's state in realtime.
+
+h2(#authenticate). Authenticate
+
+An "API key":/auth#api-keys is required to authenticate with Ably. API keys are used either to authenticate directly with Ably using "basic authentication":/auth/basic, or to generate tokens for untrusted clients using "token authentication":/auth/token.
+
+<aside data-type='important'>
+<p>The examples use "basic authentication":/auth/basic to demonstrate features for convenience. In your own applications, basic authentication should never be used on the client-side as it exposes your Ably API key. Instead use "token authentication":/auth/token. </p>
+</aside>
+
+"Sign up":https://ably.com/sign-up to Ably to create an API key in the "dashboard":https://ably.com/dashboard or use the "Control API":/account/control-api to create an API programmatically.
+
+API keys and tokens have a set of capabilities assigned to them that specify which operations, such as @subscribe@ or @publish@ can be performed on which resources. To use the Models SDK, the API key requires the following "capabilities":/auth/capabilities
+
+* @Subscribe@ for the channels you intend to subscribe to.
+* @History@ if you intend to "sync from historical messages":/livesync/models/models#history.
+
+h2(#install). Install
+
+The Models SDK requires a realtime client created using the "Ably JavaScript SDK":/getting-started/setup to interact with the Ably service.
+
+Install the Ably JavaScript SDK and the Models SDK from NPM:
+
+```[sh]
+npm install ably @ably-labs/models
+```
+
+Import the SDKs into your project:
+
+```[javascript]
+import ModelsClient from '@ably-labs/models';
+import { Realtime } from 'ably/promises';
+```
+
+Instantiate a realtime client using the Ably JavaScript SDK and pass the generated client into the Models constructor:
+
+```[javascript]
+const client = new Realtime.Promise({ key: '{{API_KEY}}' });
+const modelsClient = new ModelsClient({ client });
+```
+
+h3(#options). ClientOptions
+
+In addition to the underlying Ably realtime client, you can provide a number of other "@ClientOptions@":/api/realtime-sdk/types#client-options to configure the behavior of the Models SDK:
+
+- @syncOptions@ := is used to configure how the model state is synchronised via the Sync function.
```

Comment:
should link to sync function heading

Reply:
https://github.com/ably/docs/pull/2082/commits/c112ade0468d905e588f441cff4a895bd9b30ace